var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -74,17 +92,37 @@ def edit(self, spec, prefix):
         # make sure CC variable comes from build environment
         make_inc.filter("CC *[?]*= * .*cc", "")
 
+        make_inc.filter("LIBS *[?]*= * .*", "LIBS = " + ld_flags)
+
     @property
     def build_targets(self):
         targets = list()
 
         # use $CC set by Spack
         targets.append("CC = {0}".format(self.compiler.cc))
 
-        if self.spec.satisfies("^mkl"):
-            targets.append("MKLROOT = {0}/mkl".format(env["MKLROOT"]))
+        ld_flags = ""
+
+        if "^mkl" in self.spec:
+            targets.append("MKLROOT = {0}".format(env["MKLROOT"]))
 
-        # pass BLAS library flags
-        targets.append("LIBS = {0}".format(self.spec["blas"].libs.ld_flags))
```

Comment:
presumably this line is not needed anymore as you edited the `LIBS` above, right?

Reply:
rebased and remove